### ISSUE TITLE
checkpoint问题

### ISSUE THREAD

**Issue by kendy201124** [Azong-HQU/MMTrack#4]
你好大佬，我使用MMTrack_ep0150.pth.tar推理的时候，出现以下报错，看上去是MMTrack_ep0150.pth.tar有text_encoder.embeddings.position_ids 这个变量，但是roberta-base的模型里面并没有，我的pretrained_networks/roberta-base是从huggingface上下载直接使用的；
另外['roberta.pooler.dense.weight', 'roberta.pooler.dense.bias']这两个变量似乎没有进行训练；
请问，我是对于roberta-base的处理缺少了什么吗？

Evaluating    1 trackers on   280 sequences
Tracker: mmtrack baseline None ,  Sequence: airplane-1
test config:  {'MODEL': {'PRETRAIN_FILE': 'OSTrack_ep0300.pth.tar', 'EXTRA_MERGER': False, 'RETURN_INTER': False, 'RETURN_STAGES': [], 'BACKBONE': {'TYPE': 'vit_base_patch16_224_ce', 'STRIDE': 16, 'MID_PE': False, 'SEP_SEG': False, 'CAT_MODE': 'direct', 'MERGE_LAYER': 0, 'ADD_CLS_TOKEN': False, 'CLS_TOKEN_USE_MODE': 'ignore', 'CE_LOC': [3, 6, 9], 'CE_KEEP_RATIO': [0.7, 0.7, 0.7], 'CE_TEMPLATE_RANGE': 'CTR_POINT'}, 'TEXT_ENCODER': 'roberta-base', 'FREEZE_TEXT_ENCODER': True, 'VLFUSION_LAYERS': 1, 'VL_INPUT_TYPE': 'separate', 'DECODER': {'DEC_LAYERS': 3, 'HIDDEN_DIM': 256, 'MLP_RATIO': 8, 'NUM_HEADS': 8, 'DROPOUT': 0.1, 'VOCAB_SIZE': 1001, 'BBOX_TYPE': 'xyxy', 'MEMORY_POSITION_EMBEDDING': 'sine', 'QUERY_POSITION_EMBEDDING': 'learned'}, 'HEAD': {'TYPE': 'MLP', 'NUM_CHANNELS': 256}}, 'TRAIN': {'LR': 0.0004, 'WEIGHT_DECAY': 0.0001, 'EPOCH': 150, 'LR_DROP_EPOCH': 125, 'BATCH_SIZE': 32, 'NUM_WORKER': 2, 'OPTIMIZER': 'ADAMW', 'BACKBONE_MULTIPLIER': 0.1, 'GIOU_WEIGHT': 2.0, 'L1_WEIGHT': 5.0, 'FREEZE_LAYERS': [0], 'PRINT_INTERVAL': 50, 'VAL_EPOCH_INTERVAL': 1000, 'GRAD_CLIP_NORM': 0.1, 'AMP': True, 'BBOX_TASK': True, 'LANGUAGE_TASK': True, 'AUX_LOSS': False, 'CE_START_EPOCH': 20, 'CE_WARM_EPOCH': 50, 'DROP_PATH_RATE': 0.1, 'SCHEDULER': {'TYPE': 'step', 'DECAY_RATE': 0.1}}, 'DATA': {'SAMPLER_MODE': 'causal', 'MEAN': [0.485, 0.456, 0.406], 'STD': [0.229, 0.224, 0.225], 'MAX_SAMPLE_INTERVAL': 200, 'TRAIN': {'DATASETS_NAME': ['LASOT_Lang'], 'DATASETS_RATIO': [6], 'SAMPLE_PER_EPOCH': 60000}, 'VAL': {'DATASETS_NAME': ['GOT10K_votval'], 'DATASETS_RATIO': [1], 'SAMPLE_PER_EPOCH': 10000}, 'SEARCH': {'SIZE': 384, 'FACTOR': 5.0, 'CENTER_JITTER': 4.5, 'SCALE_JITTER': 0.5, 'NUMBER': 1}, 'TEMPLATE': {'NUMBER': 1, 'SIZE': 192, 'FACTOR': 2.0, 'CENTER_JITTER': 0, 'SCALE_JITTER': 0}}, 'TEST': {'TEMPLATE_FACTOR': 2.0, 'TEMPLATE_SIZE': 192, 'SEARCH_FACTOR': 5.0, 'SEARCH_SIZE': 384, 'EPOCH': 150}}
**Some weights of RobertaModel were not initialized from the model checkpoint at pretrained_networks/roberta-base and are newly initialized: ['roberta.pooler.dense.weight', 'roberta.pooler.dense.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
Error(s) in loading state_dict for MMTrack:
        Unexpected key(s) in state_dict: "text_encoder.embeddings.position_ids".**

**Comment by Azong-HQU** [comment]
@kendy201124 你好，我们使用的transformers库是4.12.5版本（[install.sh](https://github.com/Azong-HQU/MMTrack/blob/main/install.sh)），如果使用其他版本的库可能会出现模型参数不匹配的问题，所以可以检查一下transformers库版本是否一致。

**Comment by kendy201124** [comment]
谢谢大佬，安装4.12.5的transformers确实ok了
